weaviate/embedded.py

```diff
@@ -199,20 +202,23 @@ def ensure_running(self) -> None:
             self.start()
 
     def start(self) -> None:
+        random_ports = get_random_ports(4)
+
         self.ensure_weaviate_binary_exists()
         my_env = os.environ.copy()
 
         my_env.setdefault("AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED", "true")
         my_env.setdefault("QUERY_DEFAULTS_LIMIT", "20")
         my_env.setdefault("PERSISTENCE_DATA_PATH", self.options.persistence_data_path)
         # Bug with weaviate requires setting gossip and data bind port
-        my_env.setdefault("CLUSTER_GOSSIP_BIND_PORT", str(get_random_port()))
+        my_env.setdefault("CLUSTER_JOIN", f"localhost:{random_ports[3]}")
+        my_env.setdefault("CLUSTER_GOSSIP_BIND_PORT", str(random_ports[3]))
```

Comment:
these should probably not be the same? But this should also not be part of this PR?

Reply:
Ahh, another part of debugging that I did not undo 😅 Will remove!